test/statham_logger_test.exs

```diff
@@ -14,12 +14,48 @@ defmodule StathamLoggerTest do
         StathamLogger,
         device: :user,
         level: nil,
-        metadata: []
+        metadata: [
+          sensitive_keys: [:password],
```

Comment:
I suggest an API that is used in Phoenix for filtering params:
rename `sensitive_keys` to `filter_keys` and use `{:keep | :discard, [atom()}` api
```elixir
metadata: [
  filter_keys: {:discard, [:password]}
  # ...
]
```

Reply:
Done. I was thinking about something similar.